.travis.yml

```diff
@@ -25,15 +25,13 @@ matrix:
   - node_js: '0.10'
     env: BROWSER_NAME=ie BROWSER_VERSION=latest BROWSER_PLATFORM="Windows 2012"
   - node_js: '0.10'
-    env: BROWSER_NAME=edge BROWSER_VERSION=latest BROWSER_PLATFORM="Windows 10"
+    env: BROWSER_NAME=microsoftedge BROWSER_VERSION=latest BROWSER_PLATFORM="Windows 10"
   - node_js: '0.10'
     env: BROWSER_NAME=iphone BROWSER_VERSION=6.1
   - node_js: '0.10'
     env: BROWSER_NAME=iphone BROWSER_VERSION=7.1
   - node_js: '0.10'
-    env: BROWSER_NAME=iphone BROWSER_VERSION=8.4.1
-  - node_js: '0.10'
-    env: BROWSER_NAME=iphone BROWSER_VERSION=9.1
```

Comment:
The current version of ios is 9.x, so only `BROSER_VERSION=latest` would be required here.